tests/br_other/run.sh

```diff
@@ -95,8 +95,33 @@ fi
 
 # make sure we won't stuck in non-scheduler state, even we send a SIGTERM to it. 
 # give enough time to BR so it can gracefully stop.
-sleep 10
-! curl http://$PD_ADDR/pd/api/v1/config/schedule | grep '"disable": true' 
+sleep 5
+if curl http://$PD_ADDR/pd/api/v1/config/schedule | jq '[."schedulers-v2"][0][0]' | grep -q '"disable": false'
+then
+  echo "TEST: [$TEST_NAME] failed because scheduler has not been removed"
+  exit 1
+fi
+
+pd_settings=5
+# we need reset pd scheduler/config to default
+# until pd has the solution to temporary set these scheduler/configs.
+run_br validate reset-pd-config-as-default
+
+# max-merge-region-size set to default 20
+curl http://$PD_ADDR/pd/api/v1/config/schedule | jq '[."max-merge-region-size"]' | grep "20" || ((pd_settings--))
```

Comment:
We can query the field directly by `."max-merge-region-size"` here(and below). Why store the result to an array here...?

(`[20]` vs. `20`)

Reply:
Ok, I change it